.vib/thanos/bucketweb/cypress/cypress/integration/thanos_bucketweb_spec.js

```diff
@@ -0,0 +1,21 @@
+/// <reference types="cypress" />
+
+it('allows to see runtime & build information', () => {
+    cy.visit('/');
+    cy.contains('.dropdown-toggle', 'Status').click();
+    cy.contains('[class="dropdown-item"]', 'Runtime & Build Information').click();
+    cy.contains('Runtime Information').should('be.visible');
+    cy.get('h2').contains('Build Information').should('be.visible');
+})
+
+it('allows to see the Blocks page', () => {
+    cy.visit('/');
+    cy.get('.nav-link').contains('Blocks').click();
+    cy.contains('No blocks found.');
+})
+
+it('allows changing of the UI', () => {
+    cy.visit('/');
+    cy.contains('[class="nav-link"]', 'Classic UI').should('be.visible').click();
+    cy.contains('.nav-link', 'New UI').should('exist');
```

Comment:
Hmm I'm not really sure how Bucketweb UI works behind the scenes, but this might not guarantee to be working on a second run. If the UI is switched to the Classic one and preserved for the second run, this will fail.

Reply:
Also, I believe the `exist` is not needed here.